immersion/kubeflow_pipelines/walkthrough/solutions/lab-01_vertex.ipynb

```diff
@@ -0,0 +1,880 @@
+{
```

Comment:
<p>When I ran this, I was missing the aiplatform library. In the README where we have the requirements for KFP, we can add <code>pip install google-cloud-aiplatform</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done!